### DISCUSSION TITLE
NeighborLoader - Sample neighborhood of selected node ids

### DISCUSSION THREAD

**Discussion by dahe-cvl** [pyg-team/pytorch_geometric#4480] [Q&A]
Hi,

Is it possible to sample only selected target nodes with the corresponding neighborhood (k-hop) by using the NeighborLoader class? If yes can you give me an example?

many thanks in advance!

**Comment by rusty1s** [comment]
You can instantiate your `loader` just as usual, and then run:
```python
nodes = [0, 1, 2]
out = loader.neighbor_sampler(nodes)
data = loader.transform_fn(out)
```
